platform-hub-api/app/models/user.rb

```diff
@@ -6,6 +6,8 @@ class User < ApplicationRecord
   audited descriptor_field: :email
   has_associated_audits
 
+  before_create :build_flags
```

Comment:
This uses a method provided by the `has_one` association (figured it's easier to reuse that directly than wrap it in another method).